docs/tutorials/custom/controlling-an-intermode-rover-canbus.md

```diff
@@ -58,18 +66,20 @@ The tutorial uses the following hardware:
 - [12V to 5V Buck Converter](https://www.amazon.com/dp/B01M03288J)
 - [USB-C Male Plug to Pigtail Cable](https://www.amazon.com/Type-C-Cable-10inch-22AWG-Pigtail/dp/B09C7SLHFP)
 
-## Initial setup
+## Setup
 
-### Raspberry Pi software setup
+### Machine setup
 
-Before proceeding, [set up `viam-server` on your Raspberry Pi](/get-started/installation/prepare/rpi-setup/).
+Before you can use Viam on your device, you must ensure it has a supported operating system.
+If you are using a Raspberry pi, start by [setting up your Raspberry Pi](/get-started/installation/prepare/rpi-setup/).
+For other Single Board Computers, see the [installation guide](/get-started/installation/#prepare-your-board).
```

Comment:
```suggestion
For other single-board computers, see the [installation guide](/get-started/installation/#prepare-your-board).
```
Idk for sure if we have to capitalize if not stating "SBC"? But I think generally we say "single-board"